stable/wordpress/templates/deployment.yaml

```diff
@@ -26,11 +26,7 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: ALLOW_EMPTY_PASSWORD
-        {{- if .Values.allowEmptyPassword }}
-          value: "yes"
-        {{- else }}
-          value: "no"
-        {{- end }}
+          value: {{ .Values.allowEmptyPassword | quote }}
```

Comment:
I'm okey with this change but I think it's already quoted on `values.yaml`. I would unquote it on `values.yaml`.

Reply:
You did the opposite. I'd rather leave it as:

```
value: {{ .Values.allowEmptyPassword | quote }}
```

And correct `values.yaml` so it looks like:

```
...
allowEmptyPassword: yes
...
```

Reply:
Having `allowedEmptyPassword: yes` (without quotes) in the `values.yaml` will treat the `allowedEmptyPassword` as a boolean and result in the value `true` for the `ALLOW_EMPTY_PASSWORD` env variable.  The `bitnami/wordpress` image only accepts the value `yes/no` and hence the container launch will fail.

This if block also ensures that `ALLOW_EMPTY_PASSWORD` is assigned the correct value when users specify `--set allowedEmptyPassword=yes` on the helm command line.

Reply:
note: this is also the reason why the tests are failing in the CI (https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/charts/5550/pull-charts-e2e/9800/)